### PR TITLE
feature/More-Motion-Methods

### PR DESCRIPTION
**Pull request by BeanZip**
added left and right movement and Control Remaps

### CODE REVIEW

File: TeamCode/src/main/java/team/phantompanthers/AutoTestOpCode.java [hunk 1]
```diff
@@ -14,28 +14,29 @@ public abstract class AutoTestOpCode extends LinearOpMode implements Movement {
 
     DcMotor topLeftMotor;
```
Comment (konekowo) [inline TeamCode/src/main/java/team/phantompanthers/AutoTestOpCode.java]: Please add private to these fields. You can also remove the fields and use a hashmap that maps the motor to a string instead.

File: TeamCode/src/main/java/team/phantompanthers/ExtendableArmsAutoOpCode.java [hunk 2]
```diff
@@ -72,13 +76,28 @@ public void lowerArm(double power, long time, @NotNull DcMotor arm) {
...
+     * @param arm   -> Arm of the Robot
+     */
     @Override
     public void extendArm(double power, long time, @NotNull DcMotor arm) {
         arm.setPower(power);
         sleep(time);
     }
+
+    @Override
+    public void driveLeft(double power, long time) {
```
Comment (konekowo) [inline TeamCode/src/main/java/team/phantompanthers/ExtendableArmsAutoOpCode.java:96]: Please add documentation.

File: TeamCode/src/main/java/team/phantompanthers/ExtendableArmsAutoOpCode.java [hunk 3]
```diff
@@ -72,13 +76,28 @@ public void lowerArm(double power, long time, @NotNull DcMotor arm) {
...
+        topLeftMotor.setPower(power);
+        bottomRightMotor.setPower(power);
+        sleep(time);
+    }
+
+    @Override
+    public void driveRight(double power, long time) {
```
Comment (konekowo) [inline TeamCode/src/main/java/team/phantompanthers/ExtendableArmsAutoOpCode.java:110]: Please add documentation.

File: TeamCode/src/main/java/team/phantompanthers/PlayerTestOpCode.java [hunk 4]
```diff
@@ -13,12 +13,17 @@
  * It provides basic telemetry feedback and responds to gamepad joystick input.
  */
 @TeleOp()
-public abstract class PlayerTestOpCode extends LinearOpMode {
-    Gamepad gamepad;
-    public PlayerTestOpCode(Gamepad gamepad){
+public abstract class PlayerTestOpCode extends LinearOpMode implements Movement {
+    DcMotor topLeftMotor;
```
Comment (konekowo) [inline TeamCode/src/main/java/team/phantompanthers/PlayerTestOpCode.java]: Please add private to these fields. You can also remove the fields and use a hashmap that maps the motor to a string instead.

**Review by konekowo — CHANGES_REQUESTED** [review]
Please look at the comments I made to the changes. Once they have been fixed, request another review and I'll merge it.

**Comment by BeanZip** [comment]
ok

**Review by konekowo — APPROVED** [review]
Looks good, merging.

**Pull request merged**